src/operator/quantization/quantized_conv.cc

```diff
@@ -23,6 +23,7 @@
  * \brief
  * \author Ziheng Jiang, Jun Wu
 */
+#ifndef _MSC_VER
```

Comment:
I think we should not offer different functionality in Windows than on other systems